Write Development Tutorial

SimplyMika opened the pull request:
This is a pretty comprehensive guide on how to setup a development environment for Ace.

Note that the advice for Windows is untested as I don't have a working Windows machine. (But it's only like 3 steps and I did some research.)

I based this off of my table-of-contents branch, which I have also PR'd, let me know if I did it wrong.

Also, of course, feel free to apply any changes or ask me to change stuff, I'll be happy to!

SimplyMika committed:
Add a table of contents to README.md

SimplyMika committed:
Write comprehensive dev tutorial

SimplyMika committed:
Add Usage heading and fix formatting

SimplyMika committed:
Fix whitespace after headings

SimplyMika committed:
Remove colon from heading

SimplyMika committed:
Update Table of Contents

Run1e commented:
Thoughts on moving the installation guide in a separate markdown file? For example `INSTALL.md`?

SimplyMika committed:
Restructure tutorial

Put tutorial into INSTALL.md
Refer to INSTALL.md from README.md
Change wording and formatting here and there to make sense
